swift/Todo/TasksViewController.swift

```diff
@@ -320,15 +320,9 @@ class TasksViewController: UITableViewController, UISearchResultsUpdating, UISea
     
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let row = self.data![indexPath.row]
-        let docID = row.string(at: 0)!
-        let doc = database.document(withID: docID)!
-        
-        let complete: Bool = !doc.boolean(forKey: "complete")
-        updateTask(taskID: docID, withComplete: complete)
+        taskIDForImage = row.string(at: 0)!
         
-        // Optimistically update the UI:
-        let cell = tableView.cellForRow(at: indexPath) as! TaskTableViewCell
-        cell.accessoryType = complete ? .checkmark : .none
+        self.performSegue(withIdentifier: "showTaskDetail", sender: self)
```

Comment:
Can you keep the same behavior when tapping the row to check and uncheck? Show the detail using edit action should be enough.